example/src/main/kotlin/com.expedia.graphql.sample/query/SimpleQuery.kt

```diff
@@ -33,7 +34,9 @@ class SimpleQuery: Query {
     private fun privateFunctionsAreNotVisible() = "ignored private function"
 
     @GraphQLDescription("performs some operation")
-    fun doSomething(@GraphQLDescription("super important value") value: Int): Boolean = true
+    fun doSomething(@GraphQLDescription("super important value")
+                    @CustomDirective
```

Comment:
does this actually work? in schema gen i see we are only adding directives on functions and objects